botorch/acquisition/analytic.py

```diff
@@ -163,6 +163,23 @@ class PosteriorMean(AnalyticAcquisitionFunction):
         >>> pm = PM(test_X)
     """
 
+    def __init__(
+        self,
+        model: Model,
+        objective: Optional[ScalarizedObjective] = None,
+        maximize: bool = True,
+    ) -> None:
+        r"""Single-outcome Posterior Mean.
+
+        Args:
+            model: A fitted single-outcome GP model (must be in batch mode if
+                candidate sets X will be)
+            objective: A ScalarizedObjective (optional).
+            maximize: If True, consider the problem a maximization problem.
```

Comment:
My concern about this is that it is not clear what happens internally to consider it a maximization / minimization problem. It may be nice to make it explicit that this negates the posterior mean if False.

It may help avoid the scenario where one optimizes `PosteriorMean(..., maximize=False)` and uses the reported value as the minimum value of the posterior mean (when it is actually -1 * minimum.).

Reply:
@saitcakmak Thanks for the suggestion. If I understand you correctly, you are proposing to just change the docstring?

Reply:
Yeah, I think adding a note warning about this within the docstring should be sufficient.